x-pack/plugins/security_solution/common/api/timeline/model/api.ts

```diff
@@ -14,11 +14,13 @@ import type { Maybe } from '../../../search_strategy';
 import { Direction } from '../../../search_strategy';
 import type { PinnedEvent } from '../pinned_events/pinned_events_route';
 import { PinnedEventRuntimeType } from '../pinned_events/pinned_events_route';
+// TODO https://github.com/elastic/security-team/issues/7491
```

Comment:
Is this `TODO` still necessary?

Reply:
@michaelolo24, the Timelines API currently uses the legacy ErrorSchema from the detection engine. To decouple our API schemas, I'm considering moving it under the timeline domain. So yes, this todo is still relevant.